packages/lesswrong/components/posts/PostsPage/PostsTopSequencesNav.jsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react'
 import { registerComponent, Components } from 'meteor/vulcan:core';
+import { withRouter } from 'react-router';
 
-const PostsTopSequencesNav = ({classes, post, sequenceId}) => {
+const PostsTopSequencesNav = ({post, sequenceId, routes}) => {
   const { SequencesNavigation, CollectionsNavigation } = Components
   const canonicalCollectionSlug = post.canonicalCollectionSlug;
   const title = getNavTitle(post)
   const titleUrl = getNavTitleUrl(post)
 
-  if (sequenceId && !canonicalCollectionSlug) {
+  const isSequenceRoute = () => {
```

Comment:
There isn't actually any reason to make this a function. Just make it a Boolean and run it when you declare it.